packages/frontend-near-wallet/src/utils/constants.ts

```diff
@@ -0,0 +1,21 @@
+import Environments from '../../../../features/environments.json';
+import { NEAR_WALLET_ENV } from './config';
+
+export const WALLET_MIGRATION_VIEWS = {
+    MIGRATION_PROMPT: 'MIGRATION_PROMPT',
```

Comment:
I believe this screen will be removed to maintain neutrality of `near.org` cc @agileurbanite